activerecord/lib/active_record/core.rb

```diff
@@ -201,6 +201,15 @@ def self.current_preventing_writes
         false
       end
 
+      def self.preventing_writes?(class_name)
```

Comment:
```suggestion
      def self.preventing_writes?(class_name) # :nodoc: 
```

This is purely internal API right?